lib/validates_zipcode/cldr_regex_collection.rb

```diff
@@ -132,117 +134,120 @@ module CldrRegexpCollection
       PF: /\A987\d{2}\z/,
       PG: /\A\d{3}\z/,
       PM: /\A9[78]5\d{2}\z/,
-      PN: /\APCRN 1ZZ\z/,
+      PN: /\APCRN[ ]?1ZZ\z/i,
       PW: /\A96940\z/,
-      RE: /\A9[78]4\d{2}\z/,
-      SH: /\A(ASCN|STHL) 1ZZ\z/,
+      RE: /\A((97)(4|7|8)[0-9][0-9])\z/,
+      SH: /\A(ASCN|STHL|TDCU)[ ]?1ZZ\z/i,
       SJ: /\A\d{4}\z/,
-      SZ: /\A[HLMS]\d{3}\z/,
-      TC: /\ATKCA 1ZZ\z/,
+      SZ: /\A[HLMS]\d{3}\z/i,
+      TC: /\ATKCA[ ]?1ZZ\z/i,
       WF: /\A986\d{2}\z/,
       XK: /\A\d{5}\z/,
       YT: /\A976\d{2}\z/,
       # NOTE: UAE has no postal codes
       # https://github.com/dgilperez/validates_zipcode/issues/28
-      # AE: /\A([a-zA-Z\d\s]){3,}\z/,
+      # AE: /\A([A-Z\d\s]){3,}\z/i,
       AF: /\A\d{4}\z/,
       AL: /\A\d{4}\z/,
-      AG: /\A([a-zA-Z\d\s]){3,}\z/,
-      AO: /\A([a-zA-Z\d\s]){3,}\z/,
-      AZ: /\A([a-zA-Z\d\s]){3,}\z/,
-      BF: /\A([a-zA-Z\d\s]){3,}\z/,
-      BI: /\A([a-zA-Z\d\s]){3,}\z/,
-      BS: /\A([a-zA-Z\d\s]){3,}\z/,
-      BZ: /\A([a-zA-Z\d\s]){3,}\z/,
+      AG: /\A([A-Z\d\s]){3,}\z/i,
+      AO: /\A([A-Z\d\s]){3,}\z/i,
+      AZ: /\A([A-Z\d\s]){3,}\z/i,
+      BF: /\A([A-Z\d\s]){3,}\z/i,
+      BI: /\A([A-Z\d\s]){3,}\z/i,
+      BS: /\A([A-Z\d\s]){3,}\z/i,
+      BZ: /\A([A-Z\d\s]){3,}\z/i,
```

Comment:
This group seems to be a default code of some sort; I added the `i` to remove the `a-zA-Z`

I've also noticed when using the `random_example` today that the `\s` is allowing `\` in the zipcodes; I'm not sure if that's desired?